lib/azure/armrest/virtual_machine_image_service.rb

```diff
@@ -78,21 +75,21 @@ def skus(offer, location = @location, publisher = @publisher)
       #
       # Example:
       #
-      #   vmim.versions('14.04.2', 'UbuntuServer', 'eastus', 'Canonical')
+      #   vmim.versions('15.10', 'UbuntuServer', 'eastus', 'Canonical').map(&:name)
       #
       #   # sample output
-      #   => ["14.04.201503090", "14.04.201505060", "14.04.201506100", "14.04.201507060"]
+      #   => ["15.10.201511111", "15.10.201511161", "15.10.201512030"]
```

Comment:
The sample output is a string array. The actual return type is an array of `ImageVersion`. What does `ImageVersion` look like?